pkg/transforms/policy.go

```diff
@@ -37,16 +41,129 @@ func PolicyResourceBuilder(p *p.Policy) *PolicyResource {
 	if okns && okpp {
 		node.Properties["_parentPolicy"] = pnamespace + "/" + ppolicy
 	}
+
 	return &PolicyResource{node: node}
 }
 
+// For cert, config, operator policies.
+// This function returns `annotations`, `managedFields` for `source`,
+// and `severity`, `compliant`, `responseAction`, `parentPolicyNs`, `parentPolicyName`
+// If `parentPolicyNs`, `parentPolicyName` do not exist, the policy is a stand-alone policy
+func getPolicyCommonProperties(c *unstructured.Unstructured, node Node) Node {
+	node.Properties["annotations"] = c.GetAnnotations()
+	node.Properties["managedFields"] = c.GetManagedFields()
```

Comment:
Does everything in the Node get indexed or stored? If so, I would rather we slim down what parts of the `managedFields` we want to keep.

Reply:
Is this what you intend Could you check?

Reply:
The "isExternal" thing is pretty much what I was thinking. Thanks!